shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1086,6 +1095,57 @@ void WebContents::Invoke(bool internal,
                  std::move(callback), internal, channel, std::move(arguments));
 }
 
+void WebContents::ReceivePostMessage(const std::string& channel,
+                                     blink::TransferableMessage message) {
+  auto wrapped_ports = MessagePort::EntanglePorts(isolate(), message.ports);
```

Comment:
Should this be `std::move(message.ports )`?

Reply:
sure, that saves us a vector copy. it's semantically the same though.